extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RuleApplyPermanentViewMarker.scala

```diff
@@ -36,7 +37,11 @@ class RuleApplyPermanentViewMarker extends Rule[LogicalPlan] {
     plan mapChildren {
       case p: PermanentViewMarker => p
       case permanentView: View if hasResolvedPermanentView(permanentView) =>
-        PermanentViewMarker(permanentView, permanentView.desc)
+        val resolvedSubquery = permanentView.transformAllExpressions {
+          case scalarSubquery: ScalarSubquery =>
```

Comment:
```scala
   permanentView.transformAllExpressions {
          case s: ScalarSubquery => s.withNewPlan(PermanentViewMarker(s.plan, permanentView.desc))


How about we use `PermanentViewMarker` to make it check view privilege? In this case,

- we do not need add a new code-branch in PrivilegesBuilder
- we can fail the subqueries before executing them when view privilege failed
        }
```

Reply:
Can't pass `permanentView.desc`, will request a wrong column. Use `null` is ok?